DevGuideExamples/DCPS/Messenger/README.md

```diff
@@ -0,0 +1,27 @@
+# Messenger Example
+
+A very basic OpenDDS example, covered in the OpenDDS Developer's Guide.
+
+If you just want to build and run this, it will be built along with OpenDDS and
+can be run using `perl run_test.pl` assuming the `setenv` file has been sourced
+or OpenDDS has been installed. To have it use the DDS standard RTPS protocol
```

Comment:
This seems to state that `run_test.pl` will work without environment variables as long as OpenDDS has been installed, which I don't think is the case.

Reply:
That's true, my mistake.